lib/auth/apiserver_test.go

```diff
@@ -133,8 +133,10 @@ func TestUpsertServer(t *testing.T) {
 				require.NoError(t, err)
 				allServers = append(allServers, servers...)
 			}
+			//nolint:staticcheck // TODO(kiosion) Deprecated, remove in v20.
```

Comment:
Suggestion follow our conventional removal comment format for easier searching. Same for all other places in this PR.


```suggestion
			//nolint:staticcheck // TODO(kiosion): DELETE IN 21.0.0
```

Reply:
Gotcha, just copied an existing nolint comment.